### DISCUSSION TITLE
What the field of research will be

### DISCUSSION THREAD

**Discussion by joaopaulozz** [RensTillmann/super-forms#49] [Q&A] [answered]
Hello!!!
I have a doubt, I want to autocomplete the street by the postal code entered, can you help me?

**Comment by RensTillmann** [comment]
That's not possible, simply because google don't know that. You can't enter post code and get street name based on that. You can try this yourself just by going into google maps and entering only a zipcode, you will not get any streetname back. I have looked into possibilities of this but there is no way to do this with google places API, if you find a way to do this let me know for sure :)

**Comment by RensTillmann** [comment, marked as answer]
One thing to note though, is that you can actually grab the City, Country etc. based on the zipcode.
Just make sure you change the "The types of place results to return" on the element to be set to `(regions)`

That way you can search by zipcode. It might be wise to also limit to specific countries when doing this.

**Reply by joaopaulozz** [reply]
I got it, thanks.
